src/PageContainer.js

```diff
@@ -43,6 +52,52 @@ class PageContainer extends Component {
                 }.bind(this)
             );
         }
+
+        //TODO: query db for notifications
+        this.setState({
+            notifications: [
+                {
+                    type: 'recurring_pickup_request',
+                    content: '-L5QoXeC_UrL5tRRED3e',
+                    claimed: false
```

Comment:
It's true that we should only show notifications that haven't been acted on yet, but we'll actually handle this on the backend side, so there's no need to have `claimed` field. What will happen is:
1. When an account is notified of something, we'll add this notification obj (`{type, content}`) to `account.notifications` (this is what `this.state.notifications` will be equal to).
2. When the user performs an action on the notification that should make the notification go away, eg click on claim or reject on a pickup request, the backend will handle what should happen when the button is clicked on as well as remove it from `accounts.notifications`. We'll have a listener on the UI so that `this.state.notifications` will be updated when that happens.